crates/anchor-idl/src/program.rs

```diff
@@ -22,10 +22,16 @@ pub struct GeneratorOptions {
     pub zero_copy_unsafe: Option<PathList>,
     /// List of `repr(C)` structs.
     pub c_representation: Option<PathList>,
+    /// List of `packed(8)` structs with C repr,
+    // pub packed_8_representation: Option<PathList>,
```

Comment:
Might leave as a TODO for now, but we def want to specify `#[repr(C, packed(8)]` options

Reply:
UPDATE: Added the options